src/coreclr/jit/emit.h

```diff
@@ -3624,6 +3642,18 @@ inline unsigned emitter::emitGetInsCIargs(instrDesc* id)
             }
         }
 
+        case INS_addps:
+        {
+            if (!id->idIsEmbBroadcast())
+            {
+                return defaultSize;
+            }
+            else
+            {
+                return EA_4BYTE;
+            }
+        }
```

Comment:
This is something we'll likely want to make a bit more general purpose.

We should probably just have a general `if (id->idIsEmbBroadcast())` somewhere higher up and look up based on the instruction flags (such as the `Input_*Bit` one) before the general switch statement.